integration-tests/cucumber-tests-platform-microgrids/src/test/resources/features/osgp-adapter-ws-microgrids/ConnectionReestablish.feature

```diff
@@ -1,9 +1,8 @@
-@NightlyBuildOnly @Microgrids @Platform @Iec61850MockServerPampus
+@Microgrids @Platform @Iec61850MockServerPampus
```

Comment:
Are we going to restore the `@NightlyBuildOnly` tag at a later time? Just a question, no change needed for now.

Reply:
Yes we are. Just for testing purposes.